stripecardscan/src/main/java/com/stripe/android/stripecardscan/framework/api/StripeApi.kt

```diff
@@ -214,3 +216,71 @@ internal suspend fun uploadSavedFrames(
         errorSerializer = StripeServerErrorResponse.serializer(),
     )
 }
+
+internal fun isformatSupport(format: CardImageVerificationDetailsFormat) =
+    format == CardImageVerificationDetailsFormat.JPEG ||
+        format == CardImageVerificationDetailsFormat.WEBP
+
+internal fun imageWithConfig(
+    image: Bitmap,
+    format: CardImageVerificationDetailsFormat,
+    configs: CardImageVerificationDetailsAcceptedImageConfigs
+): Pair<ByteArray?, Rect> {
+
+    val imageSettings = configs.imageSettings(format)
+
+    // Size and crop the image per the settings.
+    val imageSize = imageSettings.imageSize!!
+    val maxImageSize = Size(imageSize.first().toInt(), imageSize.last().toInt())
+
+    val cropRect = maxImageSize
+        .scaleAndCenterWithin(image.size())
+
+    val image = image
+        .crop(cropRect)
+        .constrainToSize(maxImageSize)
+
+    // Now convert formats with the compression ratio from settings.
+    val compressionRatio = imageSettings.compressionRatio!!
+
+    // Convert to 0..100
+    val convertedRatio = compressionRatio.times(100.0).toInt()
+
+    var result: ByteArray? = null
+
+    when (format) {
+        CardImageVerificationDetailsFormat.WEBP -> {
+            result = image.toWebP(convertedRatio)
+        }
+        CardImageVerificationDetailsFormat.JPEG -> {
+            result = image.toJpeg(convertedRatio)
+        }
+    }
+
```

Comment:
nit: lift the assignment out of the when:
```suggestion
    val result = when (format) {
        CardImageVerificationDetailsFormat.WEBP -> image.toWebP(convertedRatio)
        CardImageVerificationDetailsFormat.JPEG -> image.toJpeg(convertedRatio)
    }
```